site/docs/types.md

```diff
@@ -42,6 +44,90 @@ Accepts boolean values. For example: `true`, `false`.
 
 Accepts a valid CSS color string. For example: `#f304d3`, `#ccc`, `rgb(253, 12, 134)`, `steelblue`.
 
+<br/><a name="gradient" href="#gradient">#</a> **Gradient**
+**Gradient** {% include tag ver="5.4" %}
+
+Accepts an object that specifies a gradient color pattern. To define a linear gradient based on a color scale, use a [GradientValue](#GradientValue) instead.
+
+For example:
+
+```json
+{
+  "gradient": "linear",
+  "stops": [
+    {"offset": 0.0, "color": "red"},
+    {"offset": 0.5, "color": "white"},
+    {"offset": 1.0, "color": "blue"}
+  ]
+}
+```
+
+<a name="LinearGradient" href="#LinearGradient">#</a>
+**Linear Gradient**
+
+A linear gradient interpolates colors along a line, from a starting point to an ending point. By default a linear gradient runs horizontally, from left to right. Use the _x1_, _y1_, _x2_, and _y2_ properties to configure the gradient direction. All coordinates are defined in a normalized [0, 1] coordinate space, relative to the bounding box of the item being colored.
+
+| Name          | Type                          | Description  |
+| :------------ | :---------------------------: | :------------|
+| gradient      | {% include type t="String" %} | {% include required %} The type of gradient. Use `"linear"` for a linear gradient.|
```

Comment:
You just copied the Vega docs. This won't work.

Reply:
This commit made a lot of work for me. Please carefully test your code before pushing it to my branch.